cardano-lib/testnet-template/config.json

```diff
@@ -5,7 +5,6 @@
   "ByronGenesisFile": "byron-genesis.json",
   "ConsensusMode": "PraosMode",
   "ConwayGenesisFile": "conway-genesis.json",
-  "EnableP2P": true,
```

Comment:
Just a random thought as I'm reading this again: These random changes in the flags put me off every time, am I the only one? Some versioning would be nice, it could potentially also allow the node to provide better error messages.

Reply:
Overall, the more changes in flags we have the more it can potentially cause confusion, so yes I agree that flag changes would ideally be minimized, well documented and have helpful logging as well.  We do provide a `minNodeVersion` in the config to indicate the minimum node version the bundled configs with any new changes are compatible with.  The release notes should indicate this change; I'll communicate the suggestion for logging also.